python/dgl/distributed/rpc.py

```diff
@@ -844,6 +844,55 @@ def finalize_server():
     finalize_receiver()
     print("Server (%d) shutdown." % get_rank())
 
+def fast_pull(name, id_tensor, part_id, service_id,
+              machine_count, group_count, machine_id,
+              client_id, local_data, policy):
+    """Fast-pull api used by kvstore.
+
+    Parameters
+    ----------
+    name : str
+        data name
+    id_tensor : tensor
+        data ID
+    part_id : tensor
+        partition ID of id_tensor
+    service_id : int
+        service_id of pull request
+    machine_count : int
+        total number of machine
+    group_count : int
+        total number of server inside machine
+    machine_id : int
+        current machine ID
+    client_id : int
+        current client ID
+    local_data : tensor
+        local data tensor
+    policy : PartitionPolicy
+        store the partition information
+    """
+    msg_seq = incr_msg_seq()
+    pickle_data = bytearray(pickle.dumps(([0], [name])))
```

Comment:
what does 0 mean here?

Reply:
0 means the data position. For the pull message, we only have a name metadata, so the position is fixed by 0.